.github/workflows/tf-deploy-basic.yaml

```
@@ -1,42 +1,64 @@
 on:
   workflow_call:
     inputs:
-      account_id:
+      artifact_path:
+        description: "The path of the artifacts to download"
         type: string
-        required: true
-      environment:
+        required: false
+        default: ""
+      aws_account_id:
+        description: "The AWS account ID"
         type: string
         required: true
-      feedback:
-        required: false
-        type: boolean
-        default: false
-      region:
+      aws_region:
+        description: "The AWS region"
         required: true
         type: string
-      role:
+      aws_role_name:
+        description: "The name of the role to assume with OIDC"
         type: string
         required: true
         default: cicd-iac
-      state_file:
-        required: false
+      environment:
+        description: "The environment name"
         type: string
-      state_bucket:
+        required: true
+      tf_deploy_override:
+        description: "If true, terraform apply will run regardless of branch"
+        type: boolean
         required: false
-        type: string
-      terraform_dir:
+        default: false
+      tf_dir:
         description: "The directory where the Terraform files are located"
         type: string
         required: false
         default: .
-      terraform_version:
-        description: "The Terraform version to use"
+      tf_feedback:
+        description: "Whether to post a comment on the PR with the Terraform plan"
+        required: false
+        type: boolean
+        default: false
+      tf_state_bucket:
+        description: "The name of the S3 bucket to store the Terraform state"
+        required: false
         type: string
+      tf_state_key:
+        description: "The name of the S3 key to store the Terraform state"
         required: false
-        default: 1.5.6
-      var_file:
+        type: string
+      tf_var_files:
+        description: "Comma separated list of Terraform variable files to pass to the plan command"
+        required: false
+        type: string
+      tf_vars:
+        description: "Comma separated list of Terraform variables to pass to the plan command"
         required: false
         type: string
+      tf_version:
+        description: "The Terraform version to use"
+        type: string
+        required: false
+        default: 1.5.6
```

```suggestion
        default: 1.6.4
```
but i don't really know how to manage terraform version. Should be deduced from the main.tf file in fact.

---

I personally havent tested 1.6 yet and I read there was breaking changes.